### ISSUE TITLE
Icons on the top menu have insufficient contrast against their background on mouse hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#862]
The following AI-windows mode(s) have this issue:
- Dark mode
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![9 1](https://user-images.githubusercontent.com/22068434/88346786-03754d00-ccfe-11ea-95fb-1d4c5f4b5d72.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1374.01. Please regress/close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1380.01, all 6 modes are compliant
